src/mlpack/core/cv/cv_base.hpp

```diff
@@ -149,7 +146,7 @@ class CVBase
    * numClasses parameter and a data::DatasetInfo parameter.
    */
   template<typename... MLAlgorithmArgs,
-           bool Enabled = MIE::TakesNumClasses & MIE::TakesDatasetInfo,
+           bool Enabled = MIE::TakesNumClasses& MIE::TakesDatasetInfo,
```

Comment:
Instead of the bitwise operator we should use ``&&`` here, however since the bool type is guaranteed to be only either a true or a false, it's just fine here to use the bitwise operator. Just might be out of the scope of the PR, so maybe we should fix this in another PR.

Reply:
I think this can be fixed within this PR.